### PR TITLE
Update it to luke_textui instead of  cd_drawtextui

### PR DESCRIPTION
**Pull request by tonislcs**
Update it to luka_textui instead of  cd_drawtextui

### CODE REVIEW

**Review by LukeWasTakenn — CHANGES_REQUESTED** [review]
Seems good for the most part except the blue parameter which doesn't need to be there at all since the color of the text ui is blue by default. It would also need to be a string.

File: client/client.lua [hunk 1]
```diff
@@ -518,7 +518,6 @@ function GarageBlip()
         SetBlipColour(garageBlip, 5)
         SetBlipDisplay(garageBlip, 2)
         SetBlipAsShortRange(garageBlip, true)
-
```
Comment (LukeWasTakenn) [inline client/client.lua:521]: Random, but ok

**Comment by LukeWasTakenn** [comment]
LGTM, thank you!

**Comment by tonislcs** [comment]
Pressed space by accident :D

**Pull request merged**